content/blog/kubecon-2019/index.md

```diff
@@ -0,0 +1,271 @@
+---
+title: "KubeCon NA 2019: Are you about to break Prod?"
+date: 2019-11-26
+meta_desc: "In this post, we recap and add more details to our KubeCon talk on setting up ephemeral environments. We will highlight how to acceptance test your application and infrastructure using ephemeral environments."
```

Comment:
```suggestion
meta_desc: "Acceptance Testing with Ephemeral Environments."
```

Reply:
FYI only, the SEO folks encouraged us to not get too long on titles, etc.   the "hygiene" tools like shorter titles, descs, etc.